models/cooperation.js

```diff
@@ -1,37 +1,48 @@
 const { Schema, model } = require('mongoose')
-const { FIELD_CANNOT_BE_EMPTY } = require('~/consts/errors')
+const { FIELD_CANNOT_BE_EMPTY, ENUM_CAN_BE_ONE_OF } = require('~/consts/errors')
 const { USER, OFFER, COOPERATION } = require('~/consts/models')
 const {
-  enums: { COOPERATION_STATUS }
+  enums: { COOPERATION_STATUS, PROFICIENCY_LEVEL_ENUM, SPOKEN_LANG_ENUM }
 } = require('~/consts/validation')
 
 const cooperationSchema = new Schema(
   {
-    offerId: {
+    offer: {
       type: Schema.Types.ObjectId,
       ref: OFFER,
-      required: [true, FIELD_CANNOT_BE_EMPTY('offer id')]
+      required: [true, FIELD_CANNOT_BE_EMPTY('offer')]
     },
     initiatorUserId: {
       type: Schema.Types.ObjectId,
       ref: USER,
       required: [true, FIELD_CANNOT_BE_EMPTY('initiator user id')]
     },
-    recipientUserId: {
-      type: Schema.Types.ObjectId,
-      ref: USER,
-      required: [true, FIELD_CANNOT_BE_EMPTY('recipient user id')]
+    additionalInfo: {
+      type: String,
+      minLength: [30, 'Please provide more than 30 symbols'],
+      maxLength: [1000, 'Please provide less than 1000 symbols']
+    },
+    requiredTutoringLevel: {
```

Comment:
maybe its better to name it requiredProficiencyLevel

Reply:
yeap,sounds good